gradle/package.json

```diff
@@ -5,5 +5,8 @@
     "devDependencies": {
         "jasmine": "^3.5.0",
         "mocha": "^5.2.0"
+    },
+    "dependencies": {
```

Comment:
please remove, jasmine is already under devDependencies and should be under dependencies